backend/README.md

```diff
@@ -75,6 +104,7 @@ flowchart LR
     end
     Storage <-.->|GORM lib| postgres[(PostgreSQL)]
     Endpoint -.->|GraphQL| front([Frontend])
+    postgres[(PostgreSQL)] --> |GUI managed | pgadmin([PgAdmin])
```

Comment:
If you want to include this, please change it to:
```
postgres <-.-> |GUI managed| pgadmin([PgAdmin])
```
I used dotted lines to indicate network connection and array direction to show data direction, so this fits better with the rest of the diagram.